images/manila-provisioner/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM openshift/origin-base
+
+RUN INSTALL_PKGS="atomic-openshift-manila-provisioner" && \
+    yum --enablerepo=origin-local-release install -y ${INSTALL_PKGS} && \
+    rpm -V ${INSTALL_PKGS} && \
+    yum clean all
+
+LABEL io.k8s.display-name="OpenShift Manila provisioner" \
+      io.k8s.description="This is a component of OpenShift Container Platform  which allows to provision persistent volumes using OpenStack Manila API." \
+io.openshift.tags="openshift,manila,openstack"
+
+CMD /usr/bin/manila-provisioner
```

Comment:
Since [1] accepts some CLI options, e.g., `kubeconfig`, would it make sense to use `ENTRYPOINT` instead?

I think `CMD` will be overwritten when the container is run with arguments [2].

[1] https://github.com/openshift/cloud-provider-openstack/blob/master/cmd/manila-provisioner/main.go
[2] https://docs.docker.com/engine/reference/builder/#understand-how-cmd-and-entrypoint-interact